### DISCUSSION TITLE
Add HTML option for Report Flags -f option

### DISCUSSION THREAD

**Discussion by peledies** [aquasecurity/trivy#7744] [Ideas]
### Description

Outputting an HTML report is cumbersome, especially when Trivy is installed with `homebrew`. I still haven't had any luck.

The template files can be found at `/opt/homebrew/Cellar/trivy/0.56.2/share/trivy/templates/html.tpl` when installed with homebrew, but they don't seem to work with the following command:

```
trivy image example/example:latest \
  --format template \
  --template /opt/homebrew/Cellar/trivy/0.56.2/share/trivy/templates/html.tpl \
  -o ./trivy.html
```

It would be great and more useful if the user could specify ``--format html``` and trivy would use the default html template provided to dump the html to the screen or use the `--output` flag to specify the output file. 

The user could also choose to not specify the `--output` flag and pipe that output wherever they would like.

### Target

None

### Scanner

None

**Comment by knqyf263** [comment]
`@` is missing. You can reference the document.
https://aquasecurity.github.io/trivy/v0.56/docs/configuration/reporting/#html
